resources_nlu/error_test/newLexicon.txt

```diff
@@ -5887,3 +5887,45 @@
 {"features":" {score:0.15152674515697737} ","formula":"(rdf:type dbpedia:City)","lexeme":"city","type":"ClassNoun"}
 {"features":" {score:0.07924929805861808} ","formula":"property:municipality","lexeme":"municipality","type":"DataProp"}
 {"features":" {score:0.6171889755251683} ","formula":"resource:Ludwig_Maximilian_University_of_Munich","lexeme":"ludwig maximilian university of munich","type":"NamedEntity"}
+{"features":" {score:0.02754881125994218} ","formula":"!property:market","lexeme":"market","type":"DataProp"}
```

Comment:
Do we really need to commit this all? Isn't this autogenerated DBpedia stuff for debugging?

Reply:
+1 let's stash it

Reply:
Does that file even need to be on git? We could put it into gitignore.

Reply:
Done